### PR TITLE
This commit is for adding responsive cards and media query to change …

### PR DESCRIPTION
**Pull request by emiliodeleonhdez**
Change in cards to be responsive and media query for Vane to change koder title to hotpink

### CODE REVIEW

File: p_r/koders.html [hunk 1]
```diff
@@ -52,85 +52,85 @@
     <!-- --card-- -->
     <div class="container mt-5">
         <div class="row">
-            <div class="col-4">
+            <div class="col-12 col-md-6 col-lg-4">
```
Comment (Sueenciso) [inline p_r/koders.html:55]: y en el codigo un cochinerooo

**Review by Sueenciso — CHANGES_REQUESTED** [review]
please change x ejemplo.

File: p_r/koders.html (hunk 1 above)
Comment (emiliodeleonhdez) [reply p_r/koders.html:55]: Okay Sue! 
Thanks!

**Review by Sueenciso — COMMENTED** [review]
Thanks!!

**Pull request merged**